### DISCUSSION TITLE
一部アイテムの出現率、及び階層調整について

### DISCUSSION THREAD

**Discussion by Thujopsis** [hengband/hengband#304] [Closed] [answered]
鑑定のロッド、及びスピードのロッドのレア度と出現階層を少し下げ、
出やすくしてみてはどうでしょうか。　短愚蛮怒をやっていて快適だった点がそこだったので、
一考の価値はあるかなと思います。

**Comment by iks3** [comment, marked as answer]
以下のスレッドに吸収します。

ベースアイテム(k_infoのオブジェクト群)調整案について (3.0.0α→3.0.0β)
https://github.com/hengband/hengband/discussions/300
